msteams-platform/concepts/outgoingwebhook.md

```diff
@@ -254,14 +260,21 @@ Your custom bot will need to reply asynchronously to the HTTP request from Micro
 
 ## Limitations
 
-* Custom bots do not have access to non-messaging APIs, such as team roster membership.
-* Custom bots cannot post into channels asynchronously; that is, not as a reply to a user message.
-* Although custom bots can use rich cards, they cannot leverage button actions like `imBack` or `invoke`.
+* Outgoing webhooks do not have access to non-messaging APIs, such as team roster membership.
+* Outgoing webhooks cannot post into channels asynchronously; that is, not as a reply to a user message.
+* Although outgoing webhooks can use rich cards, they cannot leverage button actions like `imBack` or `invoke`.
+
+## Samples for Outgoing webhook
+
+For sample code illustrating a outgoing webhook, see this sample on GitHub: 
 
-## Sample Custom bot 
+### Node JS
+[OfficeDev/msteams-samples-outgoing-webhook-nodejs](https://github.com/OfficeDev/msteams-samples-outgoing-webhook-nodejs)
 
-For sample code illustrating a custom bot, see our sample on GitHub: [OfficeDev/microsoft-teams-sample-custombot](https://github.com/OfficeDev/microsoft-teams-sample-custombot)
+### C#
+[OfficeDev/microsoft-teams-sample-custombot](https://github.com/OfficeDev/microsoft-teams-sample-custombot)
+Custom Bots is a previous name for outgoing webhooks.
 
-## Turn your custom bot into an app for Microsoft Teams
+## Turn your outgoing webhook into an app for Microsoft Teams
 
-If you’re ready to share your custom bot with others or make it publicly available, you can submit your bot to Microsoft Teams for consideration in the bot gallery. See the [Microsoft Teams Developer Preview interest form](https://aka.ms/microsoftteamsdeveloperpreviewinterestform) to learn more.
+If you’re ready to share your outgoing webhook with others or make it publicly available, you can submit your bot to Microsoft Teams for consideration in the bot gallery. See the [Microsoft Teams Developer Preview interest form](https://aka.ms/microsoftteamsdeveloperpreviewinterestform) to learn more.
```

Comment:
We should delete this (line 280) entirely. Converting from a custom bot / outgoing webhook into an app is a nontrivial thing.

Reply:
Done.

Reply:
Thanks for your review!